app/delegate-server/Contract/PlaceBid.purs

```diff
@@ -65,34 +79,46 @@ import HydraAuctionOffchain.Contract.Types
 import HydraAuctionOffchain.Contract.Validators (MkAuctionValidatorsError, mkAuctionValidators)
 import HydraAuctionOffchain.Lib.Json (caEncodeString)
 import JS.BigInt (fromInt) as BigInt
+import Node.Path (FilePath)
 
 placeBidL2 :: HydraNodeApiWebSocket -> BidTerms -> AppM Unit
 placeBidL2 ws bidTerms = do
   auctionInfo <- readAppState _.auctionInfo
   utxos <- readAppState _.snapshot <#> toUtxoMapWithoutRefScripts <<< _.utxo
+  { cardanoSk } <- asks _.config
   res <- runContractNullCosts do
     mkContractOutput identity $
-      placeBidL2ContractWithErrors auctionInfo bidTerms utxos
+      placeBidL2' (unwrap auctionInfo) bidTerms ws.newTx utxos cardanoSk
```

Comment:
Let's try to find a better name for `newTx` please ;-)

Reply:
Addressed in https://github.com/mlabs-haskell/hydra-auction-offchain/pull/32/commits/58a2c26187d308a4ea1e6a6332e33d5b4395c271